Resources/Prototypes/Chemistry/metabolizer_types.yml

```diff
@@ -44,3 +44,9 @@
 - type: metabolizerType
   id: Arachnid
   name: metabolizer-type-arachnid
+
+# Moffstation start
+- type: metabolizerType
+  id: Avali
+  name: metabolizer-type-avali
+# Moffstation end
```

Comment:
Put this in a new file, `.../Resources/Prototypes/_Moffstation/Chemistry/metabolizer_types.yml`